lucet-runtime/lucet-runtime-internals/src/instance/signals.rs

```diff
@@ -139,15 +144,7 @@ extern "C" fn handle_signal(signum: c_int, siginfo_ptr: *mut siginfo_t, ucontext
                 .as_mut()
         };
 
-        let trapcode = inst
-            .module
-            .lookup_trapcode(rip)
-            // if we couldn't find a code in the manifest, return an unknown trapcode that will be
-            // converted to a fatal trap when we switch back to the host
-            .unwrap_or(TrapCode {
-                ty: TrapCodeType::Unknown,
-                tag: 0,
-            });
+        let trapcode = inst.module.lookup_trapcode(rip);
```

Comment:
Great idea. I fixed this and left a note that the rip_addr_details should become signal-safe after lucet-module is complete, since it will be read-only data inside the module rather than part of the process symbol table.